### DISCUSSION TITLE
CHDMAN any good for PC game ISOs?

### DISCUSSION THREAD

**Discussion by Feathered-Serpent** [mamedev/discussions#73] [Q&A]
Hi everyone,

I wondered if I can take CHDMAN to compress ISOs of old PC games on CD or DVD to save some space. I want to keep those ISOs because the original media is sometimes a few decades old and might not be readable anymore in the future. I've read some things, that sometimes one can't really get the original file out of the CHD anymore.

Also I wonder if it's any useful for compressing (decrypted) DVD and BluRay images to save some space.

Same goes for Audio CDs. 

Very most of the ISOs have been created with DiscImageCreator.

**Comment by angelosa** [comment]
CHD (CD) are supported for both MAME and DOSBox-X at least, and assuming the disc doesn't have subchannel protection or be a multisession disc (former requires special handling from the input format, for latter cfr. https://github.com/mamedev/mame/issues/8242)
DVDs are untested (and not a good idea for "special" discs such as OG Xbox/PS2 or Macrovision protected PC games).
Blu-Ray are unsupported.
